File: Dante1/Module1.06/Zadanie6.18/main.c
#include <stdio.h>

int main()
{
    int matrixA[5][5];
    int matrixB[5][5];

    printf("Enter:\n");

    for(int i = 0; i < 5; i++)
    {
        for(int j = 0; j < 5; j++)
        {
            int check = scanf("%i",&matrixA[i][j]);
            if(check != 1)
            {
                printf("Incorrect input");
                return 1;
            }
        }
    }

    printf("Enter\n");

    for(int i = 0; i < 5; i++)
    {
        for(int j = 0; j < 5; j++)
        {
            int check = scanf("%i",&matrixB[i][j]);
            if(check != 1)
            {
                printf("Incorrect input");
                return 1;
            }
            if(matrixA[i][j] > matrixB[i][j])
            {
                continue;
            }
            else
            {
                matrixA[i][j] = matrixB[i][j];
            }
        }
    }

    for(int i = 0; i < 5; i++)
    {
        for(int j = 0; j < 5; j++)
        {
            printf("%i ",matrixA[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Dante1/Module1.06/Zadanie6.18/main.rs
fn main() {
    let mut input: String = String::new();

    let mut matrix_a: [[i32; 5]; 5] = [[0; 5]; 5];
    println!("Enter: ");
    for i in 0..5 {
        for j in 0..5 {
            input.clear();
            std::io::stdin().read_line(&mut input).expect("Err");
            matrix_a[i][j] = input.trim().parse().expect("err");
        }
    }

    let mut matrix_b: [[i32; 5]; 5] = [[0; 5]; 5];
    println!("Enter: ");
    for i in 0..5 {
        for j in 0..5 {
            input.clear();
            std::io::stdin().read_line(&mut input).expect("Err");
            matrix_b[i][j] = input.trim().parse().expect("err");

            if matrix_a[i][j] == matrix_b[i][j] {
                continue;
            }
            else {
                matrix_a[i][j] = matrix_b[i][j];
            }
        }
    }

    for i in 0..5 {
        for j in 0..5 {
            print!("{} ", matrix_a[i][j]);
        }
        println!();
    }
}
